Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-BUNDLE_VERSION = 4.5.9
+BUNDLE_VERSION ?= 4.5.9
```

Comment:
@guillaumerose This change shouldn't be part of this commit message.

Reply:
done :+1: